db/migrate/20210226102122_remove_priority_score_index_from_discovery_rules.rb

```diff
@@ -0,0 +1,6 @@
+class RemovePriorityScoreIndexFromDiscoveryRules < ActiveRecord::Migration[6.0]
+  def change
+    remove_index :discovery_rules, :priority
+    add_index :discovery_rules, [:priority, :name]
+  end
+end
```

Comment:
Let's just keep the index on priority column and not introduce a new one, just drop this migration from the patch completely.